### PR TITLE
Release/0.15.0

### PR DESCRIPTION
**Pull request by kidrahahjo**
## Description
Release version 0.15.0

### CODE REVIEW

File: contributors.yaml [hunk 1]
```diff
@@ -147,4 +147,9 @@ contributors:
     given-names: Charlotte
     orcid: "https://orcid.org/0000-0003-4915-1064"
     affiliation: "University of Michigan"
+  -
+    family-names: Kelly
+    given-names: Wang
```
Comment (bdice) [inline contributors.yaml:152]: This is backwards.

File: .zenodo.json [hunk 2]
```diff
@@ -111,6 +111,11 @@
             "affiliation": "University of Michigan",
             "name": "Charlotte Zhao",
             "orcid": "0000-0003-4915-1064"
+        },
+        {
+            "affiliation": "University of Michigan",
+            "name": "Wang Kelly",
```
Comment (bdice) [inline .zenodo.json:117]: Fix the other file and regenerate this so that the name order is correct.

**Review by bdice — APPROVED** [review]

File: changelog.txt [hunk 3]
```diff
@@ -26,12 +26,13 @@ Changed
 - ``flow.directives`` is deprecated in favor of ``flow.FlowProject.operation.with_directives`` (#309, #502).
 - All environments require a scheduler in order to submit, even in pretend mode (#533).
 - Submitting in pretend mode will show additional scheduler command information (#533).
+- Support fractional timeout value in Pythonic command line interface (#541).
```
Comment (csadorf) [inline changelog.txt:29]: I think this should be "Python", not "Pythonic", see also: https://www.computerhope.com/jargon/p/pythonic.htm

Comment (kidrahahjo) [reply changelog.txt:29]: @b-butler I cannot push to this branch, can you commit this suggestion?

**Review by csadorf — APPROVED** [review]

**Pull request merged**